usl_pipeline/usl_lib/usl_lib/storage/metastore.py

```diff
@@ -266,6 +266,31 @@ def update_scaling_done(
             {"needs_scaling": False, "feature_matrix_path": scaled_feature_matrix_path},
         )
 
+    @classmethod
+    def delete_all_for_study_area(
+        cls,
```

Comment:
Looks like this can be a `@staticmethod` because `cls` isn't used.

Reply:
Done